src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```
@@ -339,7 +339,8 @@
         </ItemGroup>
     </Target>
 
-    <Target Name="ProcessMauiAssets">
+    <Target Name="ProcessMauiAssets"
+        Condition="'$(EnableMauiAssetProcessing)' != 'false'">
```

I usually expect values to be either `true` not `true`:
```suggestion
        Condition="'$(EnableMauiAssetProcessing)' == 'true'">
```
Then if they are blank, it is `false`. In this case blank means `true`, which feels odd.

Is this a new "opt out" switch? Should it be `$(DisableMauiAssetProcessing)` and that value defaults to being blank?

---

I want the default to be true, so blank is true. Thus I think I need to check for run when "not false" which is run when "blank or true"

I am using enable as that is what most things start with? But maybe Disable is cool as well. Not sure what the "standard" for this type of disable flag is.

---

If the default is `true`, should you set it to `true` somewhere in a `.targets` file?
```xml
<EnableMauiAssetProcessing Condition=" '$(EnableMauiAssetProcessing)' == '' ">true</EnableMauiAssetProcessing>
```